artemis-server/src/main/java/org/apache/activemq/artemis/core/config/amqpBrokerConnectivity/AMQPMirrorBrokerConnectionElement.java

```diff
@@ -30,6 +30,8 @@ public class AMQPMirrorBrokerConnectionElement extends AMQPBrokerConnectionEleme
 
    boolean queueCreation = true;
 
+   boolean canForwardMessages = true;
```

Comment:
similarly I would keep the field the same sense as the option, plus the XML and config option should also use the same sense so that the defaults are more describable for both the XML and broker-properties usage. Currently someone using broker-properties would need to configure 'canForwardMessages = false' whilst someone using XML would need to configure 'no-messages-forwarding = true' to flip their actually-shared behaviour.